test/notification.js

```diff
@@ -530,35 +532,34 @@ exports.testGlobalOptOut = function(test)
 exports.testMessageWithoutLocalization = function(test)
 {
   let notification = {message: "non-localized"};
-  let texts = Notification.getLocalizedTexts(notification, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
   test.equal(texts.message, "non-localized");
   test.done();
 };
 
 exports.testLanguageOnly = function(test)
 {
-  let notification = {message: {fr: "fr"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
-  texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr");
+  let notification = {message: {en: "en"}};
+  test.equal(Utils.appLocale, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
+  test.equal(texts.message, "en");
   test.done();
 };
 
 exports.testLanguageAndCountry = function(test)
 {
-  let notification = {message: {"fr": "fr", "fr-CA": "fr-CA"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr-CA");
-  texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
+  let notification = {message: {"en": "en", "en-US": "en-US"}};
+  test.equal(Utils.appLocale, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
+  test.equal(texts.message, "en-US");
   test.done();
 };
 
 exports.testMissingTranslation = function(test)
 {
   let notification = {message: {"en-US": "en-US"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr");
+  Utils.appLocale = "fr";
```

Comment:
Since we're now assigning `Utils.appLocale` we should probably have a tearDown function for these tests which sets the value back to the default again.

Reply:
Good point, is it possible to handle it without explicit tearing down but rather using the sandboxing mechanism?

Reply:
Oh yea, that would be better. Yes please, please use `sandboxRequire` in `setUp` instead. Then we don't have to worry about setting the value back to the default again.